net_tcp.go

```diff
@@ -25,37 +25,37 @@ type (
 
 // NetTCP returns the IPv4 kernel/networking statistics for TCP datagrams
 // read from /proc/net/tcp.
-func (fs FS) NetTCP() (NetTCP, error) {
-	return newNetTCP(fs.proc.Path("net/tcp"))
+func (fs FS) NetTCP(readLimit *int) (NetTCP, error) {
```

Comment:
While its possible to move `readLimit` to `FS` and avoid this patch being a breaking change, I wasn't sure if we want to do that since the same struct is used in places where the `readLimit` (or a similar constant) is not used at all and won't make sense (`os.Open`s without any enforced limit, `FS` being used to access APIs which don't support limiting capabilities currently, etc.).